websites/S/Spinsha.re/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "ayanamy.jy126",
+      "id": "243279723487035393"
+    },
+    "service": "SpinShare",
+    "description": {
+      "en": "SpinShare is a chart sharing service for Spin Rhythm XD."
+    },
+    "url": "spinsha.re",
+    "version": "1.1.0",
+    "logo": "https://i.imgur.com/7DKJ3DZ.png",
+    "thumbnail": "https://i.imgur.com/oWVD4Pr.png",
```

Comment:
logo must be 512x512
and the thumbnail must be a promotional banner of the website or a screenshot of the website, not a screenshot of what it shows in your Discord.

Reply:
o okay